src/visGeometry/index.ts

```diff
@@ -851,7 +862,7 @@ class VisGeometry {
         this.threejsrenderer = this.createWebGL();
         parent.appendChild(this.threejsrenderer.domElement);
         parent["data-has-simularium-viewer-canvas"] = true;
-        this.setupControls();
+        this.setupControls(disableControls);
```

Comment:
I just added `setAllowViewPanning` to disable just panning in NBSV and now we are adding this wholesale method to disable all controls.

Out of scope here, but if front end implementations continue to have requirements about what controls should be allowed, maybe we should consolidate and `setupControls` should take a configuration object so the front end can be explicit about what controls it wants to have active, with current standard controls as default?